__tests__/app.test.js

```diff
@@ -135,9 +135,230 @@ describe("GET /api/reviews/:review_id", () => {
         .expect(400)
         .then((response) => {
           expect(response.body.msg).toBe(
-            "Nothing to see here. Check your request and try again."
+            "Nothing here. Check your request and try again."
           );
         });
     });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/reviews", () => {
+  describe("SUCCESS", () => {
+    it("should respond with status code 200", () => {
+      return request(app).get("/api/reviews").expect(200);
+    });
+
+    it("should respond with an array containing all of the reviews", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews).toBeInstanceOf(Array);
+          expect(reviews).toHaveLength(13);
+        });
+    });
+
+    it('should respond with an array of objects where each object has certain properties, including "comment_count"', () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews).toBeInstanceOf(Array);
+          expect(reviews).toHaveLength(13);
+          reviews.forEach((review) => {
+            expect(review).toHaveProperty("owner");
+            expect(review).toHaveProperty("title");
+            expect(review).toHaveProperty("review_id");
+            expect(review).toHaveProperty("category");
+            expect(review).toHaveProperty("review_img_url");
+            expect(review).toHaveProperty("created_at");
+            expect(review).toHaveProperty("votes");
+            expect(review).toHaveProperty("designer");
+            expect(review).toHaveProperty("comment_count");
+          });
+        });
+    });
+
+    it('should respond with an array of all the review objects ordered by date ("created_at")', () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body }) => {
+          const { reviews } = body;
+          expect(reviews).toEqual([
+            {
```

Comment:
this is a very bloated test; can be tidied up using 
`objectContaining` and `jest-sorted`